Wizard's Den RP rules file (#10449)
space-wizards/space-station-14: Resources/Server Info/RP_Rules.txt

﻿

> [color=#ff0000]You must be 16 or older to play. Users under 16 will be banned immediately and be unbanned when they are old enough.[/color]

Reviewer: I thought it's supposed to be 18 just for legal CYA?

Author: nah

> [color=#a4885c]The[/color] [color=#ffd700]Golden[/color] [color=#a4885c]Rule.[/color] Admins can disregard any and all of these rules if they deem it in the best interest of the current round, server, and/or community at large. They will of course be held fully accountable for their actions if they exercise this privilege.

Reviewer: ```suggestion
[color=#a4885c]The[/color] [color=#ffd700]Golden[/color] [color=#a4885c]Rule.[/color] Admins can disregard any and all of these rules if they deem it in the best interest of the current round, server, and/or community at large. They will of course be held fully accountable for their actions if they exercise this privilege.
    - An admin's word is law—follow it, don't argue it. If you think an admin is acting unfairly or otherwise abusing their position, you can file a complain on the forums later.
```

[color=#ff0000]Some documents available on the wiki at wiki.spacestation14.io such as Space Law, the Standard Operating Procedure, and the Alert Procedure will be mentioned here and are expected to be followed.[/color]

Core Rules

> [color=#a4885c]0.[/color] Take these rules as they are to be intended and not as they are written literally. Rule skirting and lawyering is also bannable at the admin teams discretion.

Reviewer: ```suggestion
[color=#a4885c]0.[/color] Take these rules as they are to be intended and not as they are written literally. Rule skirting and lawyering is also bannable at the admin team's discretion.
```

[color=#a4885c]1.[/color] Don’t be a dick. We’re all here to have fun. Antagonists have leeway here since ruining the round is kind of their job, but try to keep it interesting if you’re antag.

>     - Be considerate of what other players want. If everyone is dead, don’t stall the end of the round to finish your personal projects (such as building a shuttle).

Reviewer: ```suggestion
    - Be considerate of what other players want. For example, if everyone is dead don’t stall the end of the round to finish your personal projects (such as building a shuttle).
```

> [color=#a4885c]2.[/color] These are English servers. Do not speak other languages in IC or OOC, our staff cannot moderate it and you will be asked to stop.

Reviewer: ```suggestion
[color=#a4885c]2.[/color] This is an English server. Do not speak other languages in IC or OOC; our staff cannot moderate it and you will be asked to stop.
```

> [color=#a4885c]3.[/color] These servers are “Roleplay Expected”, meaning you should remain in character at all times unless speaking in OOC chat or local OOC (LOOC) chat. Basic roleplaying standards like not using emoji/emoticons or textspeak (:), :D, xD, lol, etc.) in speech, and using real names for characters, are expected to be followed.

Reviewer: Probably also want to mention OOC words like "admins", "the round", etc.

> [color=#a4885c]3.[/color] These servers are “Roleplay Expected”, meaning you should remain in character at all times unless speaking in OOC chat or local OOC (LOOC) chat. Basic roleplaying standards like not using emoji/emoticons or textspeak (:), :D, xD, lol, etc.) in speech, and using real names for characters, are expected to be followed.

Reviewer: ```suggestion
[color=#a4885c]3.[/color] These servers are “Roleplay Expected”, meaning you should remain in character at all times unless speaking in OOC chat or local OOC (LOOC) chat. Basic roleplaying standards like not using emoji/emoticons or textspeak (:), :D, xD, lol, etc.) in speech, and using real names for characters are expected to be followed.
```

> [color=#a4885c]4.[/color] Follow basic common sense–you cannot murder someone because they slipped you on a banana.

Reviewer: Actually apparently this is an en dash, not a hyphen. Using en dashes is cringe.

    - This isn’t a deathmatch, be humane.

>     - What is considered a reasonable reason to escalate conflict is listed below under ESCALATION RULES and is at the discretion of admins.

Reviewer: Might want to specially format `ESCALATION RULES` ?

This is also probably a good area to mention that a mild amount of overescalation is an IC issue.

>     - Declaring a revolution or going on strike is an antagonist only activity. “Cargonia” and other departmental revolutions are prohibited unless admin permission is obtained.

Reviewer: ```suggestion
    - Declaring a revolution or going on strike is an antagonist-only activity. “Cargonia” and other departmental revolutions are prohibited unless admin permission is obtained.
```

> [color=#a4885c]5.[/color] Erotic Roleplay (ERP) is [color=#ff0000]not allowed under any circumstance[/color]. This includes any direct or implied sexual acts as well as explicit mentions of sexual behavior.

Reviewer: ```suggestion
[color=#a4885c]5.[/color] Erotic Roleplay (ERP) is [color=#ff0000]not allowed under any circumstance[/color]. This includes any direct or implied sexual acts as well as explicit mentions of sexual behavior. Making arrangements to do it on another platform is also prohibited.
```

Reviewer: ```suggestion
[color=#a4885c]5.[/color] Erotic Roleplay (ERP) is [color=#ff0000]not allowed under any circumstance[/color]. This includes any direct or implied sexual acts as well as explicit mentions of sexual behavior. We reserve the right to post censored logs on the forums for everyone to make fun of you.
```
If you want to be really based.

[color=#a4885c]6.[/color] Harassment of other players for OOC/meta reasons is [color=#ff0000]not allowed[/color].

> 	- Annoying somebody in a round for IC/RP reasons is fine. Doing it consistently across multiple rounds for OOC reasons (“I don’t like this player” AKA metagrudging) to the point where it becomes seriously problematic for the other player(s) is [color=#ff0000]not[/color].

Reviewer: ```suggestion
	- Annoying somebody in a round for IC/RP reasons is fine. Doing it consistently across multiple rounds for OOC reasons (“I don’t like this player” AKA metagrudging) to the point where it becomes problematic for the other player(s) is [color=#ff0000]not[/color].
```

> [color=#a4885c]7.[/color] Hate speech, slurs and bigotry are not allowed.

Reviewer: ```suggestion
[color=#a4885c]7.[/color] Hate speech, slurs, and bigotry are not allowed.
```
We use Oxford commas in this household.

> 	- Intentionally seeking to demean others due to their actual or perceived race, sex, gender, sexual orientation or the like is not tolerated.

Reviewer: ```suggestion
	- Attempting to demean others due to their actual or perceived race, sex, gender, sexual orientation, age, et cetera is not tolerated.
```

> 	- Words that are closely tied to real life slurs are [color=#ff0000]not allowed[/color].

Reviewer: ```suggestion
	- Words that are closely tied to real life slurs, such as those than merely substitute a few letters, are [color=#ff0000]not allowed[/color].
```

    - Species-ism (IC or OOC bigotry against other in-game species) is also NOT ALLOWED. Not even for roleplay or because it's your "character".

> [color=#a4885c]8.[/color] Offensive/bad/out-of-character character names and descriptions put you at the mercy of administrators, from receiving brain damage to becoming valid to straight up getting banned on the spot depending on how bad they are.

Reviewer: ```suggestion
[color=#a4885c]8.[/color] Offensive/bad/out-of-character character names and descriptions put you at the mercy of administrators, and egregious cases can result in a ban.
```

>     - Character names should be something that a member of that species would reasonably have.

Reviewer: Do we even have a species naming policy / does the random name generator support species?

    - No famous real or fictional names of characters.

> [color=#a4885c]9.[/color] Knowingly using exploits, external programs, or other unintended game mechanics to gain in-game advantages or to lag/crash the server is [color=#ff0000]not allowed[/color].

Reviewer: ```suggestion
[color=#a4885c]9.[/color] Knowingly using exploits, external programs, modified clients, or other unintended game mechanics to gain in-game advantages or to lag/crash the server is [color=#ff0000]not allowed[/color].
```

> [color=#a4885c]10.[/color] Multi-keying/Alts on the same server is [color=#ff0000]not allowed[/color].

Reviewer: This doesn't really explain what multi-keying is.

	- Admins may ask players in possession of multiple alt accounts to choose one to play on and ban the other accounts.

> 	- If you are caught knowingly using alt accounts you may have your main account banned as well.

Reviewer: ```suggestion
	- If you are caught knowingly using alt accounts you may have your main account banned as well.
```
How are you ever _unknowingly_ using alt accounts?

[color=#a4885c]11.[/color] Using information gained outside the game to your advantage is [color=#ff0000]not allowed[/color]. IC and OOC are to be kept separate at all times.

> 	- Metacommunication, or communicating with other players via out of game methods (such as Discord) to gain an unfair advantage is also strictly [color=#ff0000]prohibited[/color].

Reviewer: ```suggestion
	- Metacommunication, or communicating with other players via out of game methods (such as Discord) to gain an unfair advantage is also strictly [color=#ff0000]prohibited[/color]. For example, do not mention the current round outside of the game until the round has ended.
```

>     - This is a roleplaying game, thus IC conflicts must only be resolved with IC (Local) chat. Utilizing voice chat, calling across the room, OOC, or other out-of-game communication methods to coordinate in-game actions is forbidden and will result in a [color=#ff0000]appeal-only permanent ban[/color] in most cases.

Reviewer: ```suggestion
    - This is a roleplaying game, thus IC conflicts must only be resolved with IC (Local) chat. Utilizing voice chat, OOC, or other out-of-game communication methods to coordinate in-game actions is forbidden and will result in a [color=#ff0000]appeal-only permanent ban[/color] in most cases.
```
Very weird example.

Author: its a weird example because it literally happened im fairly sure

>     - Using emotes or LOOC to bypass IC chat restrictions (accents, unconsciousness, death) is not allowed.

Reviewer: ```suggestion
    - Using emotes or LOOC to bypass IC chat restrictions (accents, unconsciousness, death, etc.) is not allowed.
```

>     - Preemptive spacing or destruction of high-risk items also falls under this rule.

Reviewer: There's nothing here about preemptively acquiring gamer loot.

Security Rules

> [color=#a4885c]12.[/color] Security is expected to use non-lethal measures unless faced with lethal force or a situation where non-lethal measures aren’t possible. Ideally, do not enter combat mode, as security stun batons do not deal damage, but still stun, when outside of combat mode.

Reviewer: Really abusing the commas in that sentence.

>     - If you still manage to kill someone as security, get them cloned immediately, and make sure their stuff is returned. Security should make their best effort to keep criminals alive and in custody.

Reviewer: ```suggestion
    - If you still manage to kill someone as security, get them cloned immediately and make sure their stuff is returned. Security should make their best effort to keep criminals alive and in custody. The time it takes to have them cloned counts towards their sentence.
```

[color=#a4885c]13.[/color] Prisoners should not be given sentences longer than 10 minutes. Certain extreme exceptions like bombers and assassins may warrant permabrigging, or in the very worst cases, execution.
    - Note that employment at a hostile corporation (i.e. traitor status) alone does not qualify someone for perma or execution.

>     - Security are required to follow Space Law. If in doubt, consult Centcom’s law team.

Reviewer: ```suggestion
    - Security are required to follow Space Law. If in doubt, consult Centcom’s law team.
    - Slightly-too-long sentences are considered an IC issue. Use common sense when considering if a sentence should be brought to admins.
```

Reviewer: You'll thank me later.

MRP Specific Rules

> [color=#a4885c]14.[/color] This server is a Roleplay Expected server. We define this as performing your assigned duties. This is a ‘do your job’ server. This means it is important to do what is expected out of your department and not what would be expected out of other departments.

Reviewer: ```suggestion
[color=#a4885c]14.[/color] This server is a "Roleplay Expected" server. We define this as pretending to actually be your character performing your character's assigned duties. This is a ‘do your job’ server; Nanotrasen would not hire the grossly incompetent/psychopathic/etc, so don't act that way.
```

Author: UGOHUGHOGUHOGHUGHUGHOGUHGOHUGOHUGHOGUHOGUHOGHGFUHGOHUGOHUGHOGUHOG

Author: im not doing these 'bikeshed the wording of an entire sentence' reviews

>     - Don’t do other peoples jobs for them. Opt in for the role you intend to play or change your job IC. Failure to do your basic duties may result in a job ban.

Reviewer: ```suggestion
    - Don’t do other people's jobs for them. Opt in for the role you intend to play or change your job by visiting the Head of Personnel. Failure to do your basic duties may result in a job ban.
```

[color=#a4885c]15.[/color] Each round exists independently of all other rounds. While friendships persisting through rounds is okay, using experiences from other rounds to give an advantage or disadvantage to other players is not permitted.

> [color=#a4885c]16.[/color] New Life Rule is in effect - Do not use information obtained while dead or from a previous life upon returning to the land of the living.

Reviewer: ```suggestion
[color=#a4885c]16.[/color] New Life Rule is in effect - Do not use information obtained while dead or from a previous life upon returning to the land of the living. Even if you are cloned, you do not remember the events culminating in your death
```

> Note: SSD characters are players who are disconnected or AFK. It is possible for players to return to SSD bodies. Players become catatonic when they take a ghost spawner role or commit suicide. Players are NOT able to return to these bodies without admin intervention.

Reviewer: This should really be a point under rule 17.

> [color=#a4885c]17.[/color] Do your best not to interact negatively with SSD/AFK players. Moving them to safety is acceptable, but killing, looting, or otherwise griefing them while they are away is not.

Reviewer: ```suggestion
[color=#a4885c]17.[/color] Do your best not to interact negatively with SSD/AFK players. Moving them to safety is acceptable; killing, looting, or otherwise griefing them while they are away is not.
```

    - If the player in question is an antag’s target or they are being secured by security, interactions to finish what is required of your duties/objectives are always acceptable.
    - Ahelping about an SSD/AFK player may grant an exception to this rule.

>     - This does not apply to players that are catatonic. Although needlessly killing or harming catatonic players is discouraged.

Reviewer: ```suggestion
    - This does not apply to players that are catatonic, although needlessly killing or harming catatonic players is discouraged.
```

[color=#a4885c]18.[/color] We now adhere to our own Standard Operating Procedure and Alert Procedure policy. These can be found at wiki.spacestation14.io.

>         Calling the shuttle to end the round faster despite there being no significant threats to the station will likely result in it being recalled.

Reviewer: ```suggestion
    - Calling the shuttle to end the round faster despite there being no significant threats to the station will likely result in it being recalled.
    - Repeatedly spamming shuttle calls/recalls can result in a ban.
```
Might al

Command Rules

> [color=#a4885c]19.[/color] The following jobs: Captain, HOP, HOS, CE, RD, CMO, and AI should ahelp when they need to leave.

Reviewer: ```suggestion
[color=#a4885c]19.[/color] The following jobs: Captain, HOP, HOS, CE, RD, CMO, and AI should ahelp when they need to stop playing. Do not play these roles if you do not expect to be able to finish the round.
```

Reviewer 2: How about QM?

>     - These roles must exhibit some competency. Lack of this can result in a job ban. Command roles are designed to lead and manage departments first and do second. They are not intended to become do-it-all members of their departments.

Reviewer: ```suggestion
    - These roles must exhibit some competency. Incompetence can result in a job ban. Command roles are designed to lead and manage departments first and do second. They are not intended to become do-it-all members of their departments.
```

    - Crew promoted to acting heads of staff must step down when an official head of staff arrives at the station. This is to prevent confusion with the chain of command when the new player arrives.

> [color=#a4885c]20.[/color] In the absence of a captain, the Head of Personnel or Head of Security must serve as acting captain or split the duties of the captain between themselves. In the absence of a captain, HoP, and HoS, the other command staff have equal standing in determining the acting captain.

Reviewer: Just make it explicitly the HoP unless there is no HoP, it'll make things more straightforward for everyone.

    - Only command staff may serve as acting captain. Acting departmental heads and regular crew may not serve as acting captain.

>     - An acting captain may fill in for decisions that require the captain’s authorization, but may not run the station and its departments, nor are they privy to the captain’s equipment or access outside of the command headset and nuke disk.

Reviewer: Not letting the acting captain secure the disk is just going to result in a lot of stealth ops victories.

Author: reread it

Antagonist and Ghost Role Policy

> [color=#a4885c]21.[/color] Antagonists exist to move the round forward and make things interesting and fun for everyone involved.

Reviewer: ```suggestion
[color=#a4885c]21.[/color] Antagonists exist to move the round forward and make things interesting and fun for everyone involved. The point isn't to "win", kill everyone else, or anything like that.
```

    - Do not spawn camp or prevent others from entering the round safely.

>     - Do what is needed for your goals. Destroying half the station when you only need to steal magboots is an example of going extremely overboard. Causing chaos in a department with a bomb is okay but destroying half the station is not

Reviewer: ```suggestion
    - Do what is needed for your goals. Causing chaos in a department with a bomb is okay, but destroying half the station when you only need to steal magboots is an example of going extremely overboard. 
```

>     - Team antag objectives take precedent over solo antag objectives.

Reviewer: ```suggestion
    - Team antag objectives take precedence over solo antag objectives.
```

    - Ghost spawner roles must follow their flavor text upon taking said role.

Escalation Rules

    - Self-defense is always allowed, to the extent of saving your own life. Preemptively attacking/maiming someone or hitting someone while they are already incapacitated is not considered self-defense.
    - Minor conflicts and scuffles are allowed, given that the wronged party has reasonable justification. If you are wronged, you are expected to handle the conflict non-lethally whenever possible, escalating in severity as the conflict continues.
    - As the defending party the rate of escalation increases with you, up to and including violence, while the instigator is always able to respond in kind.

>     - If a conflict leads to violence and either participant is incapacitated, the standing participant is expected to make an effort to treat the other, unless they have reason to believe the other was an antagonist.

Reviewer: ```suggestion
    - If a conflict leads to violence and either participant is incapacitated, the standing participant is expected to make an effort to treat the other or ensure medbay treats them, unless they have reason to believe the other was an antagonist.
```

[color=#a4885c]16.[/color] New Life Rule is in effect - Do not use information obtained while dead or from a previous life upon returning to the land of the living. Even if you are cloned, you do not remember the events culminating in your death.

[color=#a4885c]17.[/color] Do your best not to interact negatively with SSD/AFK players. Moving them to safety is acceptable; killing, looting, or otherwise griefing them while they are away is not.
    - SSD characters are players who are disconnected or AFK. It is possible for players to return to SSD bodies. Players become catatonic when they take a ghost spawner role or commit suicide. Players are NOT able to return to these bodies without admin intervention.
    - If the player in question is an antag’s target or they are being secured by security, interactions to finish what is required of your duties/objectives are always acceptable.
    - Ahelping about an SSD/AFK player may grant an exception to this rule.

>     - This does not apply to players that are catatonic, although needlessly killing or harming catatonic players is discouraged.

Reviewer 3: some version of this should be added to the original ruleset.

Author: also agree

[color=#a4885c]18.[/color] We now adhere to our own Standard Operating Procedure and Alert Procedure policy. These can be found at wiki.spacestation14.io.
    - Calling the shuttle to end the round faster despite there being no significant threats to the station will likely result in it being recalled.

[color=#ff0000]Command Rules[/color]

[color=#a4885c]19.[/color] The following jobs: Captain, HOP, HOS, CE, RD, CMO, and QM should ahelp when they need to stop playing. Do not play these roles if you do not expect to be able to finish the round.
    - These roles must exhibit some competency. Incompetence can result in a job ban. Command roles are designed to lead and manage departments first and do second. They are not intended to become do-it-all members of their departments.
    - Crew promoted to acting heads of staff must step down when an official head of staff arrives at the station. This is to prevent confusion with the chain of command when the new player arrives.

> [color=#a4885c]20.[/color] In the absence of a captain, the Head of Personnel or Head of Security must serve as acting captain or split the duties of the captain between themselves. In the absence of a captain, HoP, and HoS, the other command staff have equal standing in determining the acting captain.

Reviewer 3: ```suggestion
[color=#a4885c]20.[/color] In the absence of a captain, the Head of Personnel or Head of Security (in that order) must serve as acting captain or split the duties of the captain between themselves. In the absence of a captain, HoP, and HoS, the other command staff have equal standing in determining the acting captain.
```

    - Once treated the conflict is over; any new conflict with either individual must escalate once again. If you get into a conflict again with that individual, they may be removed permanently from the round.
    - Not all conflicts should be solved in character. If a non-antagonist conflict has escalated to the point of interfering with the round in a detrimental way, Admin Help is preferred as a solution.
    - If you’re unsure whether escalation is justified in a given conflict, don’t be afraid to use Admin Help and ask for clarification.

[color=#ff0000]Admin Interactions[/color]

    - Use ahelp (admin help) to send the admins a message when someone is breaking the rules, or a problem has occured and you need assistance.
    - Do not use ahelp to complain, or if the problem can be solved in-character.
    - When sending an ahelp, be polite and do not lie. When using ahelp to describe a situation, provide as many details as you can, such as names of people involved.
    - When getting an ahelp from an admin, take the message seriously and respond in a timely manner.

>     - An admin's word is law—follow it, don't argue it. If you think an admin is acting unfairly or otherwise abusing their position, you can file an admin complaint with the head administrator through Discord.

Reviewer 3: p o r t t h i s t o t h e s t a n d a r d r u l e s.

Author: i will!!!!!!!!!!!!!!!!!
